### ISSUE TITLE
OBJSounds doesn't seem to work with NVDA 2021.3

### ISSUE THREAD

**Issue by Futyn-Maker** [Christianlm/objsounds#4]
Hello!

I've tryed to use this addon in last alpha-version of NVDA (2021.3 in future) and found that it doesn't work. The addon absolutely breaks speech, NVDA can't speak something mindful. The log is here:
```
Traceback (most recent call last):
  File "eventHandler.pyc", line 284, in executeEvent
  File "eventHandler.pyc", line 98, in __init__
  File "eventHandler.pyc", line 107, in next
  File "D:\test\userConfig\addons\objsounds\globalPlugins\objsounds\__init__.py", line 64, in event_gainFocus
    play(obj.role)
  File "D:\test\userConfig\addons\objsounds\globalPlugins\objsounds\__init__.py", line 39, in play
    f = sounds[role]
KeyError: <Role.LISTITEM: 15>
```

Can it be fixed?

Thanks

**Comment by Christianlm** [comment]
The alpha versions are for NVDA 2022.1.
Try this draft release, seems fixes the error:
[objSounds21.11.14-dev](https://github.com/Christianlm/objsounds/releases/download/untagged-5cc25a223e7fce15728a/objsounds-21.11.14-dev.nvda-addon)

@Lo-lo78 @Futyn-Maker 

Thanks.

**Comment by Christianlm** [comment]
Sorry, the link before is for a draft release.
Download the test build from here 
<https://christianlm.github.io/files/tBuild/objsounds-21.11.14-dev.nvda-addon>


@Lo-lo78 @Futyn-Maker

Chris.

Chris ha scritto il 15/11/2021 alle 12:04:

**Comment by Christianlm** [comment]
Sorry, the link before is for a draft release.
Download the 
[testBuild](https://christianlm.github.io/files/tBuild/objsounds-21.11.14-dev.nvda-addon)

@Futyn-Maker  @Lo-lo78

Tanks.

**Comment by Futyn-Maker** [comment]
This build seems to work correctly, many thanks! You can commit this code to the main branch.
